Main.java

```diff
@@ -9,7 +11,41 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    // get the user's number and store it for later
+    System.out.println("Please enter a number to play FizzBuzz:");
+    int number = input.nextInt();
+
+    // store a variable if a number is divisable by 3
+    int Fizz = number%3;
+
+    // store a variable if a number is divisable by 5
+    int Buzz = number%5;
+
+    // store a variable if a number is divisable by 3 and 5
+    int FizzBuzz = number%3 & 5;
+
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. These should start with lowercase letters.